### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jdknives**
Updating the documentation to improve readability ease of getting started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,301 +132,6 @@ Then you can start a hypervisor with:
 $ hypervisor 
 ```
 
-### Apps
```
Comment (evanlinjin) [inline README.md:159]: Why have we gotten rid of this section?

Comment (evanlinjin) [reply README.md:159]: Also, isn't the `# Transports` and `# App programming API` sections also needed?

Comment (jdknives) [reply README.md:159]: Added the transport section back in. The API section needs to be fixed afaics.

Comment (evanlinjin) [reply README.md:159]: @jdknives yes, these need to be updated. Should we do that in another PR/ticket?

**Pull request merged**